assignments/objects.js

```diff
@@ -18,7 +18,40 @@ const example = {
 }
 
 // Write your intern objects here:
+const Mitzi = {
+  "id": 1,
+  "name": "Mitzi Melloy",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const Kennan = {
+  "id": 2,
+  "name": "Kennan Diben",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M"
+}
```

Comment:
```suggestion
speak: function() {
    return `Hello, my name is ${this.name}!`;
  }
}
```
You could have defined this here rather than below, but the way you have it is also correct. Same for Antoinetta